ui/cypress/integration/slow_query/list.spec.js

```diff
@@ -236,6 +226,364 @@ describe('SlowQuery list page', () => {
             })
         })
       })
+
+      it('Run workload without use database', () => {
+        let queryData = {
+          query: 'SELECT sleep(1.5);',
+          database: '',
+        }
+        cy.task('queryDB', { ...queryData })
+
+        cy.wait(2000)
+        cy.reload()
+        // global and use database queries will be listed
+        cy.get('[data-automation-key=query]').should('has.length', 3)
+
+        cy.get('[data-e2e=base_select_input]')
+          .click()
+          .then(() => {
+            cy.get('.ms-DetailsHeader-checkTooltip')
+              .click()
+              .then(() => {
+                // global query will not be listed
+                cy.get('[data-automation-key=query]').should('has.length', 2)
+              })
+          })
+      })
+    })
+
+    describe('Search function', () => {
+      it('Default search text', () => {
+        cy.get('[data-e2e=slow_query_search]').should('be.empty')
+      })
+
+      it('Search item with space', () => {
+        cy.get('[data-e2e=slow_query_search]')
+          .type(' SELECT sleep\\(1\\) {enter}')
+          .then(() => {
+            cy.get('[data-automation-key=query]').should('has.length', 1)
+          })
+
+        // clear search text
+        cy.get('[data-e2e=slow_query_search]')
+          .clear()
+          .type('{enter}')
+          .then(() => {
+            cy.get('[data-automation-key=query]').should('has.length', 3)
+          })
+      })
+
+      it('Type search without pressing enter then reload', () => {
+        cy.get('[data-e2e=slow_query_search]').type(' SELECT sleep\\(1\\)')
+
+        cy.reload()
+        cy.get('[data-automation-key=query]').should('has.length', 3)
```

Comment:
Maybe we want to verify that `[data-e2e=slow_query_search]` is empty?

Reply:
In the previous `it` set, I search (with enter action) `SELECT sleep(1)` and get one result, then clear the search (with enter action) text and get 3 results. 

And In this `it` set, I want to test search without pressing enter. If we type text on search box without pressing enter, then the `/slow_query/list` will not be triggered, and the length of result should still be 3.